packages/components-library/src/components/filters/StringFilter.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <b-input-group>
+    <b-form-input
+      v-model="model"
+      :name="name"
+      :placeholder="placeholder"
+      trim
+    />
+    <b-input-group-append>
+      <b-button
+        variant="outline-secondary"
+        @click.prevent="model=''"
+      >
+        <font-awesome-icon icon="times" />
+      </b-button>
+    </b-input-group-append>
+  </b-input-group>
+</template>
+
+<script>
+import Vue from 'vue'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { faTimes } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+library.add(faTimes)
+export default Vue.extend({
+  name: 'StringFilter',
+  components: { FontAwesomeIcon },
+  props: {
+    /**
+     * The HTML input element name.
+     */
+    name: {
+      type: String,
+      required: true
+    },
+    /**
+     * The HTML input element placeholder.
+     */
+    placeholder: {
+      type: String,
+      required: false,
+      default: () => ''
+    },
+    /**
+     * The StringFilter model value.
+     * @model
+     */
+    value: {
+      type: String,
+      default: () => ''
```

Comment:
Misschien deze ook required maken dan?

Reply:
Suggest to keep it , goal is to move over the components from one repo to other repo instead of updating / trimming the components